### PR TITLE
alternative to Pagination

### PR DESCRIPTION
**Pull request by celsowm**
Is there a doc page about this new class (NumericPaginator) ?

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
@ADmad

**Comment by ADmad** [comment]
@celsowm `NumericPaginator` isn't something new, paginator classes were already used indirectly through the `PaginatorComponent`. We just deprecated the middleman `PaginatorComponent`. You continue using `Controller::paginate()` method and `Controller::$paginate` property as before as explained here https://book.cakephp.org/4/en/controllers/pagination.html#pagination

Directly using paginator classes is only required in cells which is already documented https://book.cakephp.org/4/en/views/cells.html#paginating-data-inside-a-cell

File: en/controllers/components/pagination.rst [hunk 1]
```diff
@@ -7,6 +7,7 @@ Pagination
 
 .. deprecated:: 4.4.0
     The paginator component is deprecated as of 4.4.0 and will be removed in 5.0.
+    Instead you can use the :php:class:`~Cake\\Datasource\\Paging\\NumericPaginator`
```
Comment (ADmad) [inline en/controllers/components/pagination.rst]:
```suggestion
    You can use pagination from controllers as explained :doc:`here </controllers/pagination>`
```

**Comment by dereuromark** [comment]
Merge or close?

**Pull request merged**